interface/usergroup/facilities.php

```diff
@@ -188,10 +204,24 @@ function refreshme() {
                                     if ($iter3["state"]!="") {
                                         $varstate=$iter3["state"].",";
                                     }
+
+                                    $varmstreet=$iter3["mail_street"];
```

Comment:
what does the line above do?

Reply:
I don't know what those lines do, to be honest. I was following the pattern that was set before it.

Reply:
sounds good, then keep it